tests/consensus_tests/test_failed_snapshot_recovery.py

```
@@ -286,35 +290,52 @@ def test_dirty_shard_handling_with_active_replicas(tmp_path: pathlib.Path, trans
     # Wait for start of shard transfer
     wait_for_collection_shard_transfers_count(peer_api_uris[0], COLLECTION_NAME, 1)
 
+    # check that the flag was removed the when transfer was started
+    # There's a small gap between the moment the "start shard transfer" is acknowledged and the moment the shard transfer is actually initiated
+    # If we check in between, this fails so we add a sleep
+    sleep(1)
+    if transfer_method != "snapshot":
+        assert not os.path.exists(flag_path)
```

Ruff (0.8.2)

`os` may be undefined, or defined from star imports

(F405)